Heejung/Lv2/괄호 변환.js

```diff
@@ -0,0 +1,42 @@
+// u, v로 쪼개기
+function divide(p) {
+  let u = '';
+  let v = '';
+  let isCorrect = true; // 올바른 괄호인지
+
+  for (let i = 0, tmp = 0; i < p.length; i += 1) {
+    tmp += p[i] === '(' ? 1 : -1;
+    if (tmp < 0) isCorrect = false;
+    if (tmp === 0) {
+      u = p.substring(0, i + 1);
+      v = p.substring(i + 1);
+      break;
+    }
+  }
+  return [u, v, isCorrect];

```

Comment:
```suggestion
      return [u, v, isCorrect];
    }
  }
```
break를 한 뒤에 바로 return이 된다면 break 대신 return을 써줄수도 있습니다.